src/context/authentication-provider.test.tsx

```diff
@@ -0,0 +1,40 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import React from 'react';
+
+import { mount } from 'enzyme';
+import { createStore } from 'redux';
+import { reducer } from '../store/authentication';
+import { IfAuthenticated } from './if-authenticated';
+import { AuthenticationContext } from './authentication';
+import { AuthenticationContextProvider } from './authentication-provider';
+
+describe('AuthenticationProvider', () => {
+  const ChildComponent = () => {
+    return <></>;
+  };
+
+  const subject = (state: any = {}) => {
+    const store = createStore(reducer, state);
```

Comment:
as with the other place, we shouldn't need to create a store in tests. it's a lot of overhead that should be avoidable.